bench/app.py

```diff
@@ -59,15 +67,18 @@ class Healthcare(AppConfig):
 			dependencies = [{"frappe/erpnext": "~13.17.0"}]
 		"""
 		self.name = name.rstrip('/')
-		self.remote_server = "github.com"
 		self.to_clone = to_clone
 		self.on_disk = False
-		self.use_ssh = False
+		self.from_name_tag = False
+		self.remote_git_host_for_name_tag = host
+		self.use_ssh_for_name_tag = use_ssh
```

Comment:
I think we can improve the variable naming here. Maybe something like:

```suggestion
		self.is_partial_url = False
		self.git_host = host
		self.use_ssh = use_ssh
```